connect_ext_ppr/services/pricing.py

```diff
@@ -56,24 +56,63 @@ def apply_pricelist_to_marketplace(
     @returns None
     @raises ClientError, CBCClientError
     """
+    logger.info(
+        'Pricelist uploading %s_%s_%s: started.',
+        deployment_request.id,
+        marketplace.pricelist_id,
+        marketplace.marketplace,
+    )
+
     reseller_id = _identify_reseller_id(
         client=connect_client,
         batch_id=marketplace.pricelist_id,
         marketplace_id=marketplace.marketplace,
         hub_id=deployment_request.deployment.hub_id,
     )
+    logger.info(
+        'Pricelist uploading %s_%s_%s: reseller id %s.',
+        deployment_request.id,
+        marketplace.pricelist_id,
+        marketplace.marketplace,
+        reseller_id,
+    )
     excel_file, file_name, dataset = _prepare_file(
         client=connect_client,
         batch_id=marketplace.pricelist_id,
     )
+    create_dr_file_to_media(
```

Comment:
Is it just a backup file?
I suggest to write the filename construction principle in the DR article like we did for "Marketplace update" and "Delegate to L2" tasks.

Reply:
fixed